### PR TITLE
Description of functions from the package

### PR DESCRIPTION
**Pull request by jungwirt**
The only thing that has been modified is the addition of a man folder containing .Rd files for each of the functions in the miLAG package.

### CODE REVIEW

File: miLAG_package/milags_functions.R [hunk 1]
```diff
@@ -0,0 +1,1047 @@
...
+}
+
+
+
+#' get_theme (not exported)
+#'
+#' This function sets a ggplot theme without grid
+#' @param text_size defaults to 12
+#' @returns a ggplot theme
+#' @export
```
Comment (bognabognabogna) [inline miLAG_package/milags_functions.R]: it is not exported so @export should nor be there

File: miLAG_package/milags_functions.R [hunk 2]
```diff
@@ -0,0 +1,1047 @@
...
+get_theme <- function(text_size = 12) {
```
Comment (bognabognabogna) [inline miLAG_package/milags_functions.R:1005]: Can we add a short description? Even though we don;t want to export this

File: miLAG_package/milags_functions.R [hunk 3]
```diff
@@ -0,0 +1,1047 @@
...
+#' @param max_iter max. number of iterations; defaults to 100
+#' @param lower_bound lower bound for the bounded nls optimization; defaults to 0
+#' @returns lag and the nls fitting object with parameters fitted to logistic model
+calc_lag_fit_to_logistic_with_lag <- function(gr_curve, n0,
+                                                      init_gr_rate = init_gr_rate,
+                                                      init_K = init_K,
+                                                      init_lag = init_lag,
+                                                      algorithm = "auto",# Levenberg-Marquardt", # or "port" for nls
+                                                      max_iter = 100,
+                                                      lower_bound = c(0,0,0)) {
```
Comment (bognabognabogna) [inline miLAG_package/milags_functions.R]: sometimes we define functions withh each argument is a separate line, sometimes they are in the same line -> make it homogeneous

**Review by bognabognabogna — CHANGES_REQUESTED** [review]

File: miLAG_package/milags_functions.R (hunk 1 above)
Comment (jungwirt) [reply miLAG_package/milags_functions.R]: Removed export from milags_functions and NAMESPACE.

File: miLAG_package/milags_functions.R (hunk 2 above)
Comment (jungwirt) [reply miLAG_package/milags_functions.R:1005]: Done

File: miLAG_package/milags_functions.R (hunk 3 above)
Comment (jungwirt) [reply miLAG_package/milags_functions.R]: Done ( only for defining functions, mutate column values are still in multiple lines).

**Review by bognabognabogna — APPROVED** [review]

**Pull request merged**